test/dynamo/test_subclasses.py

```diff
@@ -1363,9 +1363,9 @@ def binary(nt1, nt2):
 
     # TODO: cannot parametrize this test class with device for some reason
     def _test_autograd(self, backend):
-        a = torch.randn(2, 3, requires_grad=True, dtype=torch.float64)
-        b = torch.randn(3, 3, requires_grad=True, dtype=torch.float64)
-        c = torch.randn(4, 3, requires_grad=True, dtype=torch.float64)
+        a = torch.randn(2, 5, requires_grad=True, dtype=torch.float64)
+        b = torch.randn(3, 5, requires_grad=True, dtype=torch.float64)
+        c = torch.randn(4, 5, requires_grad=True, dtype=torch.float64)
```

Comment:
5 so that the 3 from batch size isn't reused, right? :p

Reply:
I think so yeah haha